### PR TITLE
Add HDR InfiniBand Arkouda perf testing

### PR DESCRIPTION
**Pull request by ronawho**
We have testing on 56 Gbps FDR IB currently. Add some additional testing
on newer 200 Gbps HDR IB.

For https://github.com/Cray/chapel-private/issues/1490

### CODE REVIEW

File: util/cron/test-perf.cray-cs-hdr.arkouda.bash [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+CWD=$(cd $(dirname $0) ; pwd)
+
+export CHPL_TEST_PERF_CONFIG_NAME='16-node-cs-hdr'
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.cray-cs-hdr.arkouda"
+
+# setup arkouda
+source $CWD/common-arkouda.bash
+export ARKOUDA_NUMLOCALES=16
+
+module list
```
Comment (e-kayrakli) [inline util/cron/test-perf.cray-cs-hdr.arkouda.bash:14]: is there some voodoo behind this, or is it just to make it easier to debug a failure?

**Review by e-kayrakli — APPROVED** [review]
I am wondering if we should stick `ib` somewhere in the config name. Otherwise, looks good!

File: util/cron/test-perf.cray-cs-hdr.arkouda.bash (hunk 1 above)
Comment (ronawho) [reply util/cron/test-perf.cray-cs-hdr.arkouda.bash:14]: No voodoo, just debugging -- https://github.com/chapel-lang/chapel/pull/16678

**Comment by ronawho** [comment]
> I am wondering if we should stick ib somewhere in the config name

I was wondering about that too, but for now I figured keep it closer to the other CS configs and change it if/when we add CS testing with other networks.

**Pull request merged**